packages/core/src/html/MdAttributeRenderer.ts

```diff
@@ -53,24 +62,27 @@ class MdAttributeRenderer {
    * @returns {boolean} whether the node has both the slot and attribute
    */
   // eslint-disable-next-line class-methods-use-this
-  hasSlotOverridingAttribute(node, attribute, slotName = attribute) {
+  hasSlotOverridingAttribute(node: DomElement, attribute: string, slotName = attribute): boolean {
     const hasNamedSlot = node.children
       && node.children.some(child => getVslotShorthandName(child) === slotName);
+    const nodeAttribs = node.attribs ?? {};
```

Comment:
Should we just do an early return here? Because if the node has no attributes, there should be nothing to process